internal/github/github.go

```diff
@@ -75,6 +96,19 @@ func (r *RealGitHub) Clone(output io.Writer, workingDir string, fullRepoName str
 	return execInstance.Execute(output, workingDir, "gh", "repo", "clone", fullRepoName)
 }
 
+func (r *RealGitHub) GetPrStatus(output io.Writer, workingDir string) (*PrStatus, error) {
+	s, err := execInstance.ExecuteAndCapture(output, workingDir, "gh", "pr", "view", "--json", "title,state,mergeable,statusCheckRollup,reviewDecision,reactionGroups,url")
```

Comment:
I might be wrong, but I'm not sure it's working on repos you've forked.
In your example it looks like you tested it on repos you own, so the `gh` client would push as a branch.

In my try on a repo I don't own, the `gh pr view` is not returning anything, however it does if i add my username at the end:
`gh pr view username:branch_name`

I stumbled upon that because I'm looking at a similar thing for the auto-close PRs thing.

I think I'm gonna generalise a GetPR function from which we can extract the number from.

Reply:
The status though will return all the PRs which are owned by the username. We can prob filter from that with the branch name (which is forced to be the campain name).

Reply:
Ended up using `gh pr status` which changes slightly the signature of the json returned.
https://github.com/Skyscanner/turbolift/pull/72/files#diff-ba9a81d402c41754e4e18c09693a602ff36de97f5e42e105a0a613201844e6efR128

Reply:
Yikes, good catch!

> I think I'm gonna generalise a GetPR function from which we can extract the number from.

Absolutely, let's get #72 merged then, and then we could amend this one to use your improved GetPR function?

Reply:
Done